employees/templates/employees/report_list.html

```diff
@@ -52,8 +52,34 @@ <h1>{{ UI_text.PAGE_TITLE.value }}</h1>
                     {{ date }}
                 </td>
                 {% for report in reports %}
-                    {% if forloop.counter > 1 %}
-                    <tr>
+                    {% if not forloop.last %}
+                        {% if forloop.counter > 1 %}
+                        <tr>
+                        {% endif %}
+                        <td class="project-column">
+                            {{ report.project }}
+                        </td>
+                        <td class="work-hours-column">
+                            {{ report.work_hours_str }}
+                        </td>
+                        <td class="description-column">
+                            {{ report.markdown_description }}
+                        </td>
+                        <td class="edit-button-column Invisible hidden-print">
+                            {% if report.editable %}
+                            <a href="{% url 'custom-report-detail' pk=report.id %}" class="btn btn-info">
+                                <span class="glyphicon glyphicon-pencil"></span>
+                            </a>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    {% else %}
+                <tr>
+                    <td style="width:230px"><strong>{{ UI_text.HOURS_PER_DAY_LABEL.value }}</strong></td>
```

Comment:
I think setting `width` this way won't work, it will get aligned to other cells - or other cells will be aligned to this one.